<launch>

	<group ns="pole_test">
		<node pkg="laser_loc" name="bot_loc" type="bot_loc" output="screen"/>
		<node pkg="laser_loc" name="get_poles" type="get_poles" output="screen"/>
	</group>

</launch>